plugins/blocks/browse/block.tpl

```diff
@@ -28,7 +28,7 @@
 				</li>
 			{/if}
 
-			{if $browseCategories}
+			{if $browseCategories->count}
```

Comment:
It's a [DAO](https://github.com/pkp/omp/blob/master/plugins/blocks/browse/BrowseBlockPlugin.inc.php#L99-L113), not an array. Happy to hear of a better way to handle this.